src/bsp/src/stm32/include/SpiHeader.h

```diff
@@ -24,10 +24,11 @@ namespace EspHeader {
         uint8_t rawValue;
     };
 
-    struct Header {
+    struct __attribute__((__packed__)) Header {
```

Comment:
Why do we need the header packed here?

Reply:
To make sure  the compiler does not pad the struct. During development, I needed to add it since the compiler was padding it up to 10 bytes for some reason